### PR TITLE
Fixes Fermi plushie reaction. Removes carbonation of caramel

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request
Makes fermi plushie need caramel rather then sugar
Lowers the heating needed

## Why It's Good For The Game

This reaction is unable to be done without this fix
Removes caramel carbonation

## Changelog
:cl:
fix: Fermi plushie can be made once again
/:cl:

### CODE REVIEW

**Comment by YakumoChen** [comment]
this actually doesn't fix it and if anything you're just making it worse. Why would you add a sugar reaction to chemistry without asking @Thalpy if it fucks anything?

File: code/modules/food_and_drinks/recipes/food_mixtures.dm [hunk 1]
```diff
@@ -84,14 +84,6 @@
...
 	mob_react = FALSE
 
-/datum/chemical_reaction/caramel_burned
-	name = "Caramel burned"
-	id = "caramel_burned"
-	results = list("carbon" = 1)
-	required_reagents = list("caramel" = 1)
-	required_temp = 483
-	mob_react = FALSE
-
```
Comment (Ghommie) [inline code/modules/food_and_drinks/recipes/food_mixtures.dm:94]: This is a legit challenge and makes chemistry a little more interesting tbh, and you already fixed the plushie reaction, no need to remove it.

**Review by Ghommie — CHANGES_REQUESTED** [review]

File: code/modules/food_and_drinks/recipes/food_mixtures.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/modules/food_and_drinks/recipes/food_mixtures.dm:94]: Yes need to remove it, this brakes more reactions like ELS and one other im told

Comment (Ghommie) [reply code/modules/food_and_drinks/recipes/food_mixtures.dm:94]: From what I have seen searching around, the only reaction this beaks is `/datum/chemical_reaction/fermi/secretcatchem` (weird how the unit test never reported of this issue).
You may want to replace the "sugar" with "astrotame" (or another sweet equivalent) on the aforementioned chemical reaction. Or make the caramel reaction require a little bit of water.

Fermi-chems are a bit of a wack to deal with. But do not remove this, you bum.

**Review by silicons — DISMISSED** [review]

**Review by Ghommie — CHANGES_REQUESTED** [review]
My previous review stands. the sekrit cat chem reaction will still be broken even if you removed the burned caramel reaction.

**Comment by Trilbyspaceclone** [comment]
Read/test the code again. This works - Edit we were talking about two different reactions. Simple mix up on my part

**Review by Ghommie — APPROVED** [review]

**Comment by Ghommie** [comment]
I dislike how the bunch of you can't really stand up for a culinary/chemistry challenge and Fermis' interest for sweeties. But consider this merged.

**Pull request merged**